.github/ISSUE_TEMPLATE/feature_request.yml

```
@@ -0,0 +1,29 @@
+name: Feature request
+description: Request a new feature
+title: "[Feature]: "
+labels: [enhancement]
+assignees:
+  - 
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this feature request!
```

```suggestion
        Thanks for taking the time to request a feature! ^^
```
im friendly like that